.editorconfig

```
@@ -18,7 +18,7 @@ indent_style = space
 indent_size = 4
 
 # YAML requires indentation with spaces instead of tabs.
-[*.{yml,yaml}]
+[*.{yml,yaml,clang{-format,-tidy,d}}]
```

Nitpick, but I think its clearer if we keep actual yaml extensions separate from the clang stuff. And it prevents globbing `whatever.clang-format`.
```suggestion
[*.{yml,yaml},.clang{-format,-tidy,d}]
```